### PR TITLE
Remove call invoker related classes

### PR DESCRIPTION
**Pull request by WyriHaximus**

### CODE REVIEW

**Comment by CharlotteDunois** [comment]
@WyriHaximus 🏓 What's the current state?

**Comment by WyriHaximus** [comment]
@CharlotteDunois It's a rabbithole I need to get back into, got some stuff locally but there are a lot of changes and EIO is being an issue.

**Comment by CharlotteDunois** [comment]
@WyriHaximus status?

**Comment by WyriHaximus** [comment]
@CharlotteDunois nothing really changes, going for a stab at it again tonight

**Comment by clue** [comment]
The less code, the better :+1: @WyriHaximus Can you add the appropriate labels (BC break etc.)?

File: src/Eio/Adapter.php [hunk 1]
```diff
@@ -253,11 +219,18 @@ protected function processLsContents($basePath, $result, ObjectStream $stream)
             ];
             $promises[] = \React\Filesystem\detectType($this->typeDetectors, $node)->then(function (NodeInterface $node) use ($stream) {
                 $stream->write($node);
+
+                return \React\Promise\resolve(true);
+            }, function ($error) {
+                echo (string)$error;
```
Comment (CharlotteDunois) [inline src/Eio/Adapter.php]: Forgotten debug code

Comment (WyriHaximus) [reply src/Eio/Adapter.php]: Removed it :+1:

**Comment by WyriHaximus** [comment]
> The less code, the better  @WyriHaximus Can you add the appropriate labels (BC break etc.)?

@clue done :+1:

**Comment by CharlotteDunois** [comment]
@jsor 🏓

**Comment by jsor** [comment]
@WyriHaximus Tests are failing ❌

**Comment by WyriHaximus** [comment]
@jsor thought about it merging anyway and fixing that in a follow up PR where it is clear where the issues resides. This also lets @CharlotteDunois continue with https://github.com/reactphp/filesystem/pull/69

**Pull request merged**